ShenkinStore/Controllers/TransactionsController.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using ShenkinStore.Models;
+
+namespace ShenkinStore.Controllers
+{
+    public class TransactionsController : Controller
+    {
+        private readonly ShenkinContext _context;
+
+        public TransactionsController(ShenkinContext context)
+        {
+            _context = context;
+        }
+
+        // GET: Transactions
+        public async Task<IActionResult> Index()
+        {
+            var shenkinContext = _context.Transaction.Include(t => t.Product).Include(t => t.User);
```

Comment:
If I understand correctly it's transactions and not shenkinContext right?
If so please change the variable name.

Reply:
Updated.